numpy/ma/core.pyi

```diff
@@ -1113,9 +1113,9 @@ def putmask(a, mask, values): ...
 def transpose(a, axes=...): ...
 def reshape(a, new_shape, order=...): ...
 def resize(x, new_shape): ...
-def ndim(obj): ...
+def ndim(obj: ArrayLike) -> int: ...
```

Comment:
in theory this seems to accept any object, but `np.ndim` is typed to accept `ArrayLike` so I've gone with that. That seems more useful that `object`

Reply:
`np.ndim` is implemented as

```py
try:
    return a.ndim
except AttributeError:
    return asarray(a).ndim
```

so this seems like the right choice to me.